### PR TITLE
azure functions walkthrough

### PR DESCRIPTION
**Pull request by esweet431**
- adds the azure functions walkthrough discussed last week
- add a new command invocation to open the walkthrough

### CODE REVIEW

File: resources/walkthroughs/scenarios.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+| If you want to...           | Then...                                                                                                                                                                   |
+|-----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Build a web API             | Implement an endpoint for your web applications using the [HTTP trigger](https://learn.microsoft.com/azure/azure-functions/functions-bindings-http-webhook)               |
+| Run scheduled tasks         | Execute code on [pre-defined time intervals](https://learn.microsoft.com/azure/azure-functions/functions-bindings-timer)                                                  |
+| Build a serverless workflow | Crete an event-driven workflow from a series of functions using [durable functions](https://learn.microsoft.com/azure/azure-functions/durable/durable-functions-overview) |
```
Comment (bwateratmsft) [inline resources/walkthroughs/scenarios.md:5]:
```suggestion
| Build a serverless workflow | Create an event-driven workflow from a series of functions using [durable functions](https://learn.microsoft.com/azure/azure-functions/durable/durable-functions-overview) |
```

**Review by bwateratmsft — DISMISSED** [review]

**Comment by nturinski** [comment]
Just an aside, but love that your GitHub profile pic is your Animal Crossing villager ☺️

**Review by alexweininger — COMMENTED** [review]
Should we make the links used in the walkthrough aka.ms shortened links?

File: resources/walkthroughs/scenarios.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
+| If you want to...           | Then...                                                                                                                                    |
+|-----------------------------|--------------------------------------------------------------------------------------------------------------------------------------------|
+| Build a web API             | Implement an endpoint for your web applications using the [HTTP trigger](https://aka.ms/functions-getstarted-httptrigger)                  |
+| Run scheduled tasks         | Execute code on [pre-defined time intervals](https://aka.ms/functions-getstarted-timertrigger)                                             |
+| Build a serverless workflow | Create an event-driven workflow from a series of functions using [durable functions](https://aka.ms/functions-getstarted-durablefunctions) |
+| Process file uploads        | Run code when a file is uploaded or changed in [blob storage](https://aka.ms/functions-getstarted-blogstorage)                             |
```
Comment (bwateratmsft) [inline resources/walkthroughs/scenarios.md:6]:
```suggestion
| Process file uploads        | Run code when a file is uploaded or changed in [blob storage](https://aka.ms/functions-getstarted-blobstorage)                             |
```

**Review by bwateratmsft — DISMISSED** [review]

**Review by alexweininger — DISMISSED** [review]

**Review by bwateratmsft — APPROVED** [review]

**Pull request merged**